integr8ly/cloud-resource-operator
# MGDAPI-4719 bump CRO dependency to prepare for OSD 4.12

## Overview
Bump CRO dependency to support changes to other operators that RHOAM consumes to keep k8s api and controller runtime version in line. 

Jira: https://issues.redhat.com/browse/MGDAPI-4719

Goal was to bump deps and clear out the replaces section as it got very tangled and unnecessary. 

## Verification
- Verify that the e2e pass
- Verify that the unit test pass
- Verify CRO can provision AWS resources as before
